plugins/vli/fu-vli-usbhub-firmware.c

```diff
@@ -178,8 +178,9 @@ fu_vli_usbhub_firmware_parse (FuFirmware *firmware,
 			g_prefix_error (error, "failed to get 820/822 byte: ");
 			return FALSE;
 		}
-		/* VL820/VL822 == VT3518 */
+		/* VL819~VL822 == VT3518 */
 		if (tmp == 0xF0) {
+			/* VL819 to be added */
```

Comment:
It seems like this needs to be updated since `FU_VLI_DEVICE_KIND_VL822` doesn't exist anymore